docs/frontend-system/architecture/04-plugins.md

```diff
@@ -10,84 +10,151 @@ description: Frontend plugins
 
 ## Introduction
 
-<!--
+In addition to the existing [`plugins`](../../plugins/index.md) documentation let's take a look at what changes for plugins in the new frontend system. If you already created a plugin yourself you will recognise a lot of similarity in the new architecture with the existing one.
 
-Introduce frontend plugins and how each of them is shipped as a separate package. Highlight that each package should only export a single plugin instance.
+Backstage is a single-page application composed of a set of plugins. Each of this plugins should solve exactly one responsibility & follow our suggested [plugin package structure & naming](../../architecture-decisions/adr011-plugin-package-structure.md).
```

Comment:
I think "solve exactly one responsibility" might cause more questions than it answers tbh. Might be worth be worth switch it to something like "Each of these plugins have have solutions for their own set of responsibilities, and should follow ..."